nixos/modules/installer/cd-dvd/installation-cd-graphical-base.nix

```diff
@@ -8,6 +8,16 @@ with lib;
 {
   imports = [ ./installation-cd-base.nix ];
 
+  # Whitelist wheel users to do anything
+  # This is useful for things like pkexec
```

Comment:
Maybe should add a caution here that this is explicitly dangerous, and should only be used for livecds. ... Just in case it gets copy-pasted somewhere.